templates/layout.html

```diff
@@ -17,15 +17,15 @@
 
 <header>
   <h1><a href=/>Neomad</a></h1>
-  <a href=# class=mobile-nav>Menu</a>
+  <button class=mobile-nav>Menu</button>
   <nav class=mobile-close>
     <a href={{ url_for('article_list') }} title="Read Neomad's adventures">Read</a>
     <a href={{ url_for('around') }}>Around me</a>
     {% if current_user and current_user.is_authenticated %}
-    <div id=avatar-menu class=avatar>
+    <div id=avatar-nav class=avatar-nav>
```

Comment:
`id=avatar-nav class=avatar-nav`??
Reminder: you would use `class` if it's a reusable behavior; you would use `id` if the element is unique per page.
I think you did this to be pointed by the JS but it still doesn't make sense to me.

If we have only 1 per page, let's keep the id.
If we may use this behavior, let's keep the class and when refering to this specific item with a nested selector.